_locales/en/messages.json

```diff
@@ -683,7 +683,7 @@
         "message": "Disarm motors after set delay [seconds] (Requires MOTOR_STOP feature)"
     },
     "configurationDisarmKillSwitch": {
-        "message": "Disarm motors regardless of throttle value (When arming via AUX channel)"
+        "message": "Arm/Disarm motors regardless of throttle value when using AUX channel"
```

Comment:
what do you think of this one : 

Main message : `Disarm motors regardless of throttle value (When ARM is configured in Modes tab via AUX channel)`
Help/hint : `Arming is disabled when the throttle is not low. Be careful as you could disarm accidentally with a switch while flying`